src/pages/User/login/index.tsx

```diff
@@ -91,7 +100,9 @@ const Login: React.FC = () => {
               <span className={styles.title}>Ant Design</span>
             </Link>
           </div>
-          <div className={styles.desc}>Ant Design 是西湖区最具影响力的 Web 设计规范</div>
```

Comment:
This was actually already defined in our translations, it looks like we might not have been using it though:

<img width="628" alt="Screen Shot 2021-01-12 at 6 45 13 PM" src="https://user-images.githubusercontent.com/463175/104388033-5dfda200-5506-11eb-87d4-e6b765730240.png">